test/test-applications/integrations/Samples.Owin.WebApi/Controllers/HomeController.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace Samples.Owin.WebApi.Controllers
+{
+    [RoutePrefix("")]
+    public class HomeController : ApiController
+    {
+        [Route("")]
+        [HttpGet]
+        public IHttpActionResult Index()
+        {
+            return Ok();
+        }
```

Comment:
For simplicity, do we want to just share sources with the existing ASP.NET WebApi2 Sample? Just a thought, not passionate about it, but would ensure we test all the same code paths :)